src/compiler/utilities.ts

```diff
@@ -26,20 +26,6 @@ namespace ts {
         return undefined;
     }
 
-    export function findDeclaration<T extends Declaration>(symbol: Symbol, predicate: (node: Declaration) => node is T): T | undefined;

```

Comment:
would it be fast enough to have `getDeclarationOfKind` delegate to `findDeclaration` ?